core/templates/pages/about-page/about-page.component.html

```diff
@@ -1,258 +1,703 @@
+    padding-top: 2vw;
+  }
+  .oppia-about-page-section-six {
+    background-color: white;
+    clear: both;
+    margin: 0 auto;
+    max-height: 60vh;
+    max-width: 100%;
+  }
+  .oppia-about-page-section-six .oppia-about-section-six-left {
+    width: 40vw;
+  }
+  .oppia-about-page-section-six .oppia-about-outcome-title {
+    color: #0f0f0f;
+    font-family: "Capriola", Arial, sans-serif;
+    font-size: 30px;
+    margin-left: 50vw;
+    padding-top: 22vh;
+  }
+  .oppia-about-page-section-six .oppia-about-outcome-content {
+    color: #121313;
+    font-family: "Roboto", Arial, sans-serif;
+    font-size: 15px;
+    margin-left: 50vw;
+    padding-top: 2vh;
+    width: 35vw;
+  }
+  .oppia-about-page-section-six .oppia-about-section-six-right {
+    background-color: white;
+    background-image: url(/assets/images/splash/dsk_testimonial_background.png);
+    height: 75vh;
+    margin: 0;
   }
-  @media only screen and (max-width: 500px) {
-    .oppia-about-page-container .oppia-about-otter-speech-card {
-      width: 50%;
+  .oppia-about-page-section-six .about-study-image {
+    float: left;
+    height: 60vh;
+    padding: 5vh;
+    padding-top: 15vh;
+  }
+  .oppia-about-section-six-right .oppia-about-classroom-button-container {
+    margin-left: 50vw;
+    padding-top: 2vw;
+  }
+  .oppia-about-page-section-seven {
+    background-color: white;
+    clear: both;
+    margin: 0 auto;
+    max-height: 30vh;
+    max-width: 100%;
+    padding-top: 5vh;
+    text-align: center;
+  }
+  .oppia-about-page-section-seven .get-started-title,
+  .oppia-about-page-section-seven .oppia-about-classroom-button-container-one,
+  .oppia-about-page-section-seven .oppia-about-classroom-button-container-two,
+  .oppia-about-page-section-seven .oppia-about-classroom-button-container-three {
+    padding-bottom: 2vh;
+    padding-top: 2vh;
+  }
+  .oppia-about-classroom-button-container-three .oppia-about-button {
+    background-color: white;
+    border-color: teal;
+    color: teal;
+  }
+  .about-page .oppia-about-page-section-four {
+    background-color: white;
+    background-image: url(/assets/images/splash/dsk_testimonial_background.png);
+  }
+  .oppia-about-page-section-four .oppia-about-button,
+  .oppia-about-page-section-five .oppia-about-button,
+  .oppia-about-page-section-six .oppia-about-button {
+    margin-top: 15px;
+  }
+  .about-page .oppia-about-button {
+      background-color: #015c53;
+      border-radius: 0;
+      color: #ffffff;
+      font-family: "Capriola", "Roboto", Arial, sans-serif;
+      font-size: 16px;
+      margin-top: -20px;
+      min-width: 250px;
+      padding: 10px;
+      text-transform: uppercase;
+    }
+    .about-page .oppia-about-button:hover,
+    .about-page .oppia-about-button:focus,
+    .about-page .oppia-about-button:active,
+    .about-page .oppia-about-bottom-button:hover,
+    .about-page .oppia-about-bottom-button:focus,
+    .about-page .oppia-about-bottom-button:active {
```

Comment:
Is the way to simplify this somehow?

Reply:
it doesn't applies as expected in simplified form, I tried.